packages/firebase_ml/lib/src/firebase_model_manager.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+
+part of firebase_ml;
+
+/// Manages the remote models.
+///
+/// Before using a remote model, download it with [FirebaseModelManager].
+/// The model name is the key for a model. The model name should be consistent with the name of the model that has been uploaded to the Firebase console.
```

Comment:
Try to keep to 80 char per line

https://dart.dev/guides/language/effective-dart/style#avoid-lines-longer-than-80-characters

Reply:
Done